RoomLocator/RoomLocator.Api/Controllers/ScadadataController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using RoomLocator.Data.Services;
+using RoomLocator.Domain.ViewModels;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Newtonsoft.Json;
+
+
+namespace RoomLocator.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    [ApiController]
+    public class ScadadataController :Controller
+    {
+        private readonly ScadadataService _service;
+        public ScadadataController(ScadadataService service)
+        {
+            _service = service;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<ScadadataViewModel>>> GetSensorData()
```

Comment:
Vi har vel ikke brug for de rå værdier i frontenden? Det er jo derfor vi laver et api til det